stable/mission-control/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # JFrog Mission-Control Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.1.1] - Jul 24, 2020
```

Comment:
I think this justifies a minor bump. There are a couple of breaking changes here

Reply:
Done